cpp/include/cudf/strings/wrap.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+namespace strings
+{
+/**
+ * @brief Inserts new-line characters (ASCII 0x0A) into each string in place of spaces, depending on `width`.  
+ *
+ * Returns a column of strings where, for each string row in the input, 
```

Comment:
```suggestion
 * For each string row in the input column longer than `width`, the corresponding output string row 
 * will have newline characters inserted so that each line is no more than `width characters`. Attempts to 
 * use existing white space locations to split the strings, but may split non-white-space sequences if necessary.
```

Reply:
Done.